Neos.Flow/composer.json

```diff
@@ -14,18 +14,18 @@
         "ext-reflection": "*",
         "ext-xml": "*",
 
-        "neos/cache": "~7.3.0",
-        "neos/eel": "~7.3.0",
-        "neos/error-messages": "~7.3.0",
-        "neos/utility-arrays": "~7.3.0",
-        "neos/utility-files": "~7.3.0",
-        "neos/utility-mediatypes": "~7.3.0",
-        "neos/utility-objecthandling": "~7.3.0",
-        "neos/utility-opcodecache": "~7.3.0",
-        "neos/utility-schema": "~7.3.0",
-        "neos/utility-unicode": "~7.3.0",
-        "neos/flow-log": "~7.3.0",
-        "neos/http-factories": "~7.3.0",
+        "neos/cache": "self.version",
+        "neos/eel": "self.version",
+        "neos/error-messages": "self.version",
+        "neos/utility-arrays": "self.version",
+        "neos/utility-files": "self.version",
+        "neos/utility-mediatypes": "self.version",
+        "neos/utility-objecthandling": "self.version",
+        "neos/utility-opcodecache": "self.version",
+        "neos/utility-schema": "self.version",
+        "neos/utility-unicode": "self.version",
+        "neos/flow-log": "self.version",
+        "neos/http-factories": "self.version",
```

Comment:
This change was lost with the release on 29.06: ee09e21b88c3ca6044df93d5e823b6b1fc66352c. @kdambekalns did you expect this to be changed during releases?

Reply:
No, didn't expect that… 😥

Reply:
Ah, the release scripts… I did not think of those. 🤦‍♂️